common/changes/@microsoft/rush/fix-404-error-handler_2024-06-11-04-10.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "When deploy new packages, the publish flow didn't detected error message correctly",
```

```suggestion
      "comment": "Fix an issue where not published packages are not correctly identified as not published when querying a package feed under certain versions of NPM.",
```

Is this specifically under certain versions of npm?